### ISSUE TITLE
Search result "displaycond" not found in current docs

### ISSUE THREAD

**Issue by Alagts** [TYPO3-Documentation/t3docs-search-indexer#99]
Currently in the TCA reference:
A search for “displaycond” in the TCA reference does not find any results. Instead a search in all docs reveals results for “displaycond” in the TCA reference. That is strange. [Link to the search results.](https://docs.typo3.org/search/search?scope=&q=displaycond). [Link to TCA reference](https://docs.typo3.org/m/typo3/reference-tca/13.4/en-us/Index.html#start). 
![displayCond_Search_Results](https://github.com/user-attachments/assets/24179137-0b40-4034-b9c4-c322865a47c9)
We need to investigate why this problem occurs.

**Comment by kanow** [comment]
Yes, I can confirm this for extension documentations. e.g. search for "configuration" in news extension. Nothing found.

![image](https://github.com/user-attachments/assets/ac402057-7bf3-47d4-b799-275eff93a775)

Empty result, and out of the news docs.

![image](https://github.com/user-attachments/assets/ae2326d9-8a8d-4879-a4bf-70ed2fbeab56)

**Comment by damienalexandre** [comment]
It looks like it's working now?

Searching "Configuration" in the news docs: https://docs.typo3.org/search/search?scope=%2Fp%2Fgeorgringer%2Fnews%2Fmain%2Fen-us%2F&q=Configuration

![image](https://github.com/user-attachments/assets/ddc3b6c0-5b9c-43ab-a904-0ff488895b49)

Searching "displayCond" in the TCA manual: https://docs.typo3.org/search/search?scope=%2Fm%2Ftypo3%2Freference-tca%2Fmain%2Fen-us%2F&q=displayCond

![image](https://github.com/user-attachments/assets/25c0b9f7-331b-4cfb-878e-af54d7eb9e58)

Let me know if I'm missing something, thanks!

**Comment by linawolf** [comment]
@Alagts can this be closed?

**Comment by linawolf** [comment]
Resolved with #110
